pandas/tests/indexes/multi/test_indexing.py

```diff
@@ -183,6 +183,155 @@ def test_get_indexer():
         idx1.get_indexer(idx2)
 
 
+def test_get_indexer_and_fill():
+    """ test getting an indexer for another index using the backfill method """
+    mult_idx_1 = MultiIndex.from_product([[0], [0, 2, 3, 4]])
+    mult_idx_2 = MultiIndex.from_product([[0], [1, 3, 4]])
+
+    tm.assert_almost_equal(mult_idx_1.get_indexer(mult_idx_2), np.array([-1, 2, 3]))
+    tm.assert_almost_equal(
+        mult_idx_1.get_indexer(mult_idx_2, method="backfill"), np.array([1, 2, 3])
+    )
+    for method in ("bfill", "backfill"):
+        tm.assert_almost_equal(
+            mult_idx_1.get_indexer(mult_idx_2, method=method), np.array([1, 2, 3])
+        )
+    for method in ("ffill", "pad"):
+        tm.assert_almost_equal(
+            mult_idx_1.get_indexer(mult_idx_2, method=method), np.array([0, 2, 3])
+        )
```

Comment:
separate lines

result = mult_idx...
expected = np.array...
tm.assert_almost_equal(result, expected)

Reply:
done (here and elsewhere), good idea; thanks